pkg/connector/client/slack.go

```diff
@@ -68,16 +69,14 @@ func NewClient(
 	}
 
 	return &Client{
-		baseUrl:                  baseUrl0,
-		baseScimUrl:              baseScimUrl0,
-		token:                    token,
-		enterpriseID:             enterpriseID,
-		botToken:                 botToken,
-		ssoEnabled:               ssoEnabled,
-		scimVersion:              finalScimVersion,
-		wrapper:                  uhttp.NewBaseHttpClient(httpClient),
-		workspacesNameCache:      make(map[string]string),
-		workspacesNameCacheMutex: sync.RWMutex{},
```

Comment:
if we release this we'll no longer be able to ship this connector via C1 and we'll not be able to update the connector in C1 anymore. So in case deploying to lambda becomes a problem we should create a C1 PR to make it so C1 will use the new separate connectors in that case or revert these changes.

Reply:
this is fine actually